projects/igniteui-angular/src/lib/calendar/calendar.component.ts

```diff
@@ -37,6 +36,20 @@ export enum CalendarSelection {
     RANGE = 'range'
 }
 
+export enum DateRangeType {
```

Comment:
These should not belong to the calendar component. They should be put in `core`.